src/main/java/com/algorand/algosdk/v2/client/Utils.java

```diff
@@ -59,7 +58,7 @@ public static DryrunRequest createDryrun(AlgodClient client, List<SignedTransact
      * @param latest_timestamp The latest timestamp the dryrun should include
      * @param round            The agreement round or block height the dryrun should
      *                         include
-     * @return TransactionResponse of the confirmed transaction
+     * @return DryrunRequest of the confirmed transaction
```

Comment:
`@return DryrunRequest to be submitted to the TealDryrun endpoint`

Reply:
jeeze ben